src/components/EventItem.js

```diff
@@ -50,7 +50,12 @@ const EventItem = (props) => {
 					>
 						<strong>{event.datetime_start}</strong>
 					</div>
-					{venue && <div>Venue: {venue}</div>}
+					{venue && (
+						<div>
+							{venue}
+							<span className="dashicons dashicons-location"></span>
+						</div>
+					)}
```

Comment:
Let's move this after the title (line 64) so it goes
DATE TIME
TITLE
VENUE

Reply:
@mauteri 
Done